.coderabbit.yaml

```diff
@@ -0,0 +1,40 @@
+# yaml-language-server: $schema=https://storage.googleapis.com/coderabbit_public_assets/schema.v2.json
+language: en
+reviews:
+  profile: chill
+  request_changes_workflow: false # Do not approve after comments resolved
+  high_level_summary: true
+  review_status: true
+  collapse_walkthrough: true
+  changed_files_summary: true
+  sequence_diagrams: true
+  related_issues: true
+  related_prs: true
+  suggested_labels: true
+  auto_apply_labels: false
+  suggested_reviewers: true
+  poem: false
+  labeling_instructions: []
+  path_filters: []
+  path_instructions: []
+  auto_review:
+    enabled: true
+    auto_incremental_review: true
+    ignore_title_keywords: []
+    drafts: true
+    base_branches:
+      - master
+  finishing_touches:
+    docstrings:
+      enabled: true
+
+chat:
+  auto_reply: true
+
+knowledge_base:
+  learnings:
+    scope: global # Organization-wide learnings
```

Comment:
Let's keep this `local`?

Reply:
Resolved in [4f4244d](https://github.com/PrairieLearn/PrairieLearn/pull/11337/commits/4f4244dabff9190d6411769f7b0bdfa6f2e8e999).